javascriptv3/example_code/dynamodb/scenarios/dynamodb_basics/src/dynamodb_basics.js

```diff
@@ -1,40 +1,30 @@
 /* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 
-ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript (v3),
 which is available at https://github.com/aws/aws-sdk-js-v3.
 
 Purpose:
 This scenario demonstrates how to:
     - Create a table that can hold movie data.
     - Write movie data to the table from a sample JSON file.
-    - Put, get, and update a single movie in the table.
-    - Update a movie.
-    - Query for movies that were released in a given year.
+    - Put, get, update, and delete a single movie in the table.
     - Scan for movies that were released in a range of years.
-    - Delete a movie from the table.
+    - Query for movies that were released in a given year.
     - Delete the table.
 
 
 Running the code:
-1. Update the AWS Region in '../libs/ddbClient.js'.
-2. Run the following at the command line:
-
-   node dynamodb_basics.js <table_name> <newMovieName> <newMovieYear> <existingMovieName> <existingMovieYear> <newMovieRank> <newMoviePlot>
-
-   For example, node dynamodb_basics.js myNewTable myMovieName 2022 'This Is the End' 2013 200 'A coder cracks code...'
+1. Change directories to the same directory as this file
+(javascriptv3/example_code/dynamodb/scenarios/dynamodb_basics/src).
+2. Import this file as a module and then call `main()`. This can be done from
+another file, or with the following command:
```

Comment:
This can be done from another file, or with the following command: --> Do this from another file, or use the following command: